src/platform/datapath_raw.c

```diff
@@ -182,6 +182,7 @@ CxPlatDataPathInitialize(
                 CxPlatSockPoolUninitialize(&DataPath->SocketPool);
             }
             CXPLAT_FREE(DataPath, QUIC_POOL_DATAPATH);
+            CxPlatRundownRelease(&CxPlatWorkerRundown);
```

Comment:
Don't you want to `rundownRelease` in all error paths?

Reply:
No. That's the bug. If `DpRawInitialized` then it will eventually be released when `CxPlatDpRawUninitialize` complete (possibly asynchronously).